src/Backend/Modules/Blog/Engine/Model.php

```diff
@@ -232,63 +234,55 @@ public static function deleteCategoryAllowed(int $id): bool
         );
     }
 
-    /**
-     * Deletes one or more comments
-     *
-     * @param array $ids The id(s) of the items(s) to delete.
-     */
     public static function deleteComments(array $ids): void
     {
-        // make sure $ids is an array
-        $ids = (array) $ids;
-
-        // loop and cast to integers
-        foreach ($ids as &$id) {
-            $id = (int) $id;
-        }
+        $repository = BackendModel::get('doctrine.orm.default_entity_manager')
```

Comment:
it would be best to create a service for this repository

Reply:
fixed in cd9cef81ef540cf410bc31750966a00591aa06ae